minimal.js

```diff
@@ -212,6 +228,14 @@ module.exports = function (dirname, keys, opts) {
           queue: []
         }
       }
+      if (!state.feeds[keys.id]) {
+        state.feeds[keys.id] = {
+          id: null,
+          timestamp: null,
+          sequence: 0,
+          queue: []
+        }
+      }
```

Comment:
@christianbundy I added this to the initialisation of validation state, it is the replacement of your case where you're looking for the state of your empty feed ... this is the one case where we know for sure that a feed is empty

Reply:
I think this makes the behavior inconsistent, since `getFeedState()` is meant to return the state of any feed passed. After this change it returns different results for an empty feed depending on whether it's your feed ID.

Reply:
yes but your feed is the only one you can authoritatively know is empty.
If you ask for the state of another feed and it's not in the `db.last` index all you can say is "I don't know what the state is". That could mean "it's empty" or "I haven't got it yet". Returning a result which suggests you know it's empty is dangerous IMO

Reply:
discussed with Christian on phone. 
we decided to remove the feedId arg from the method, and to add this method back to the manifest

Reply:
i.e. it will only get your current feed state